pkgs/servers/geospatial/geoserver/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, unzip, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "geoserver";
+  version = "2.21.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/geoserver/GeoServer/${version}/geoserver-${version}-bin.zip";
+    sha256 = "sha256-UCr22Ffhnux6eA0w5qoaf5Hvuypsl/FGpK+emi8G0Mc=";
+  };
+
+  sourceRoot = ".";
+  nativeBuildInputs = [ unzip makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeWrapper ];
```
I don't think that unzip is used

---

It's needed for unpacking, otherwise:
```
geoserver> do not know how to unpack source archive /nix/store/f9k2xn91f4zp6a7hsmr2kxpip57jk02v-geoserver-2.21.0-bin.zip
```